tuftool/src/common.rs

```diff
@@ -0,0 +1,28 @@
+/// This module if for code that is re-used by different `tuftool` subcommands.
```

Comment:
```suggestion
/// This module is for code that is re-used by different `tuftool` subcommands.
```

Reply:
Addressed in [59899dd](https://github.com/awslabs/tough/compare/3e858e5b5d629576a0c13f3f7b627cd8c49af9a4..59899ddf735ceb7e4edc883a9b1c6a195506f820),